docs/workforce-management-framework/upstream-1/master.adoc

```diff
@@ -5,34 +5,41 @@ include::topics/shared/attributes.adoc[]
 :toclevels: 5
 :last-update-label!:
 
+image::../../shared/images/logo-blue.png[logo]
+
 [[Raincatcher]]
 = {WFM-RC-NameLong}
```

Comment:
this seems to break asciidoctor when run , would be safe to remove 
``` [[Raincatcher]]
 [[Raincatcher]]
 = {WFM-RC-NameLong} 
```

the title as it is covered by the image.